_posts/2020-10-20-3.8.0-release-announcement.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp 3.8.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.8.0!
+
+## Updates to Django settings
+
+* The Django `admin` site URL is now configurable using the new `ADMIN_SITE_URL` settings parameter [7637](https://pulp.plan.io/issues/7637).
+
+* As a follow up to [#7179](https://pulp.plan.io/issues/7179), which provided a default `DJANGO_SETTINGS_MODULE` variable, in this release, the default `DJANGO_SETTINGS_MODULE` variable is now set everywhere. This simplifies configuration because the **systemd** service no longer has to set this value [#7720](https://pulp.plan.io/issues/7720).
+
+## Allowed Content Checksums Validation
+
+In the previous release `ALLOWED_CONTENT_CHECKSUMS` variable was introduced so that users can specify and limit unwanted checksums in their environment.
+
+In this release, validation has been added so that if a user has Artifacts in their database and adds checksums to `ALLOWED_CONTENT_CHECKSUMS` that do not match the existing Artifacts, the error `ImproperlyConfigured` is thrown [#7487](https://pulp.plan.io/issues/7487).
+
+## Dynaconf Version Requirement
+
+Earlier versions of dynaconf proved buggy and error prone. In this release, the required dynaconf version is 3.1.2. [#7682](https://pulp.plan.io/issues/7682).
+
+## Bug Fixes
+
+* When a task was cancelled, it was logging as a failed task [#4559](https://pulp.plan.io/issues/4559)
+* When a task was cancelled, the status of all incomplete "progress reports" was remaining in an in-progress state [#4921](https://pulp.plan.io/issues/4921)
+* A fix has been added to handle duplicate content during synchronization and migration from Pulp 2 to Pulp 3 [#7147](https://pulp.plan.io/issues/7147)
+* Content streaming has been enabled for the `RepositoryVersionDistribution` parameter [#7568](https://pulp.plan.io/issues/7568)
+* The `urlize_quoted_links` filter in Django Rest Framework was dropped because Django's `urlize` filter does the same action [#7634](https://pulp.plan.io/issues/7634)
+* The `plugin_template` was missing the `check_manifest` option that checks for files omitted from `MANIFEST.in` [#7634](https://pulp.plan.io/issues/7634)
```

Comment:
Did you mean to point to this one https://pulp.plan.io/issues/7656?

I think it can move to a feature for plugin writers or not include it at all. It's more an improvement on the plugin template, just the changelog entry appeared for pulpcore which just applied changes and added a configuration. https://github.com/pulp/plugin_template/commit/b04f54a0d351035385ad249fc8c6bdcd85dfbed7
Plugin template now contains a check-manifest script which is run in travis and which is checking if all expected files are included in the  MANIFEST.in.

I see how it can be confusing to read the changelog for pulpcore, apologies.

Reply:
Very bad copy paste of issues, this evening. Thanks for catching these. 

I added a Plugin template section.